edx_proctoring/templates/proctored_exam/instructions.html

```diff
@@ -8,13 +8,31 @@ <h3>
     {% endblocktrans %}
     </h3>
     {% if backend_instructions %}
-      {% for instruction in backend_instructions %}
-      <p>{{instruction|safe}}</p>
-      {% endfor %}
       <p>
-        <a href="#" id="software_download_link" data-action="click_download_software" target="_blank">{% trans "Start System Check" %}</a>
+        {% blocktrans %}
```

Comment:
Is the intention that this changed language only applies to backends that define instructions? (in other words, the software_secure backend will still show the old instructions, and the old button/link)

Reply:
Yes, you're reading the code correctly